Result.xcodeproj/project.pbxproj

```diff
@@ -472,6 +477,11 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Checkouts/Box/build/Debug",
+					"$(PROJECT_DIR)/Carthage/Checkouts/Box/build/Debug-iphoneos",
+				);
```

Comment:
I suspect these two are the Mac framework & tests and should just be Debug, not Debug-iphoneos.